taskSeven.java

```diff
@@ -0,0 +1,12 @@
+// napisat' class kotoruy vuvodit v consol' tri randomno sgenerirovanuh chisla bol'she 0 i men'she 100
+
+class taskSeven {
+    public static void main(String[] args) {
+        int numOne = (int) (1 + Math.random()*99);
+        int numTwo = (int) (1 + Math.random()*99);
+        int numThree = (int) (1 + Math.random()*99);
+        System.out.println(numOne);
+        System.out.println(numTwo);
+        System.out.println(numThree);
```

Comment:
You don't need to convert to int, Math.random() returns double, use it instead.
Also you can put a value of random func in output like that: 
System.out.println(1 + Math.random()*99);